README.md

```diff
@@ -1,9 +1,21 @@
 # comixcraft - Showcase
 In this repository a showcase of the comixcraft project is saved.
-Currently it is just a placeholder-page. When implementing the placeholder can be replaced by any page, of any chosen technology.
+This website presents the comixcraft editor with its key functionalities and team members. 
+
+## Run the project
+To run the project, double-click on the index.html file to open it in your default web browser or open it with a live server. 
+
+## Style Guide
+
+The font used for the **comixcraft**-Showcase is _Nunito_, it is the default for everything.
+Any text in the comics themselves is in the font _Pangolin_.
```

Comment:
You could also add the link to the fonts (https://github.com/comixcraft/Editor/tree/dev/public/fonts)